<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>536</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>578</width>
    <height>495</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="4" column="0">
    <widget class="QTextEdit" name="textEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>425</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>140</width>
       <height>425</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="image.qrc">:/C:/Users/ASUS/Pictures/ImageMaterials/qqshow.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widgetchoose" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t1.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t2.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_3">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t3.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t3.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_4">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t4.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t4.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_5">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t5.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t5.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_6">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t6.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t6.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_7">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t7.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t7.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_8">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t8.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t8.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_9">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t9.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t9.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_10">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="image.qrc">
            <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t10.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t10.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>182</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButton_11">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="image.qrc">
          <normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t11.png</normaloff>:/C:/Users/ASUS/Pictures/ImageMaterials/t11.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>373</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonclose">
          <property name="minimumSize">
           <size>
            <width>72</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>72</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSend">
          <property name="minimumSize">
           <size>
            <width>63</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>63</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonMore">
          <property name="minimumSize">
           <size>
            <width>26</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>26</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string>∨</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
